src/java.desktop/share/classes/javax/swing/border/EtchedBorder.java

```diff
@@ -119,6 +122,22 @@ public EtchedBorder(int etchType, Color highlight, Color shadow)    {
         this.shadow = shadow;
     }
 
+    private void paintBorderHighlight(Graphics g, Color c, int w, int h, int stkWidth) {
+        g.setColor(c);
+        g.drawRect(stkWidth/2, stkWidth/2, w-(2*stkWidth), h-(2*stkWidth));
+    }
+
+    private void paintBorderShadow(Graphics g, Color c, int w, int h, int stkWidth) {
+        g.setColor(c);
+        g.drawLine((3*stkWidth/2), h-(3*stkWidth/2), (3*stkWidth/2), (3*stkWidth/2)); // left line
```

Comment:
what do  you want to be the order of operations here
((3*stkWidth)/2) or (3*(stkWidth/2) ?
Make it clear with parens.